### ISSUE TITLE
What is the batchsize when training on MINST ?

### ISSUE THREAD

**Issue by terigenbuaa** [feng-bao-ucsf/DVIB#1]
The code is well written and very readable. But I still encountered lots of problems when I try to reproduce your experiment because my lack of experience in this field. 

So what is the batchsize when training? And how to implement the gradient descent part with this `DVIBTrainer` class ?

Or it would be really helpful to share your experiment code to me.  My email: teruun@163.com

Thanks in advance and best wishes!

**Comment by terigenbuaa** [comment]
Sorry I missed some information from the readme which is said this code is based on Multi-View Information Bottleneck
